lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3139,7 +3139,9 @@ namespace {
           // instead of checking if they come from the `std` module.
           if (!d->getDeclName().isIdentifier())
             return false;
-          return d->getName() == "abs" || d->getName() == "div";
+          return d->getName() == "abs" || d->getName() == "div" ||
```

Comment:
There are more of such functions in libc++, I wonder if we should do an audit and make all of them unavailable in Swift. This is not urgent, just a thought for the future.

Reply:
Yes. If we can mark them unavailable in Swift in libc++, that sounds like a much better way to do this.

Reply:
Exit is another one